code/procs/antagonist_procs.dm

```diff
@@ -221,6 +221,7 @@
 		synd_mob.equip_if_possible(new /obj/item/clothing/suit/space/syndicate/commissar_greatcoat(synd_mob), synd_mob.slot_wear_suit)
 		synd_mob.equip_if_possible(new /obj/item/device/radio/headset/syndicate/leader(synd_mob), synd_mob.slot_ears)
 		synd_mob.equip_if_possible(new /obj/item/katana_sheath/nukeop(synd_mob), synd_mob.slot_l_hand)
+		synd_mob.equip_if_possible(new /obj/item/remote/nuke_summon_remote(synd_mob), synd_mob.slot_l_store)
```

Comment:
I don't know why, but when I spawned in as the commander during local testing, the bomb remote was NOT in my left pocket. I had all the other commander dressing though.
![image](https://user-images.githubusercontent.com/9563541/92687053-ac4c2b80-f2ef-11ea-9eb7-e26d6e7bce8a.png)

Reply:
yeah, i've moved it to a hand spawn instead.